components/ScrollToTop.tsx

```diff
@@ -0,0 +1,34 @@
+import React, { useEffect, useState } from 'react';
+import { FaArrowUp } from 'react-icons/fa';
+import { Link } from 'react-scroll';
+
+function BackToTopButton() {
+  const [btnVisiblity, setBtnVisiblity] = useState(false);
+  const handleScroll = () => {
+    window.scrollTo({ top: 0, behavior: 'smooth' });
+  };
+  useEffect(() => {
+    const toggleVisiblity = () => {
+      window.pageYOffset > 250 ? setBtnVisiblity(true) : setBtnVisiblity(false);
```

Comment:
`pageYOffset` is deprecated so use `scrollY` 

```suggestion
      window.scrollY > 250 ? setBtnVisiblity(true) : setBtnVisiblity(false);
```